src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -986,7 +986,13 @@ void WorldSession::HandlePlayerLoginFromDB(LoginQueryHolder const& holder)
     bool firstLogin = pCurrChar->HasAtLoginFlag(AT_LOGIN_FIRST);
     if (firstLogin)
     {
-        pCurrChar->RemoveAtLoginFlag(AT_LOGIN_FIRST);
+        {
```

Comment:
Remove bracket

Reply:
currently in the process of rebuilding my db and need to drop a few thigns off at a warehouse. when i am back i will test with the brackets and without the brackets.

Reply:
done